testsuite/forge/src/interface/node.rs

```diff
@@ -49,10 +61,7 @@ pub trait Node {
     fn clear_storage(&mut self) -> Result<()>;
 
     /// Performs a Health Check on the Node
-    fn health_check(&self) -> Result<()>;
-
-    /// Performs a connectivity check on this Node
-    fn connectivity_check(&self) -> Result<()>; // Maybe have this as a function as a part of a validator or fullnode
+    fn health_check(&mut self) -> HealthStatus;
```

Comment:
This makes sense, although we probably want to change the health_check function for the local node to never panic